README.md

```diff
@@ -60,3 +95,44 @@ Then use the `analyticsClient` variable:
 ```js
 const urlParamsHandler = new analyticsClient.AnalyticsUrlParams();
 ```
+
+## Configuration
+
+We use environment configuration on the [analytics-backend](https://github.com/balena-io/analytics-backend) to mask the actual API tokens of the final destination services of these events.
+We expose easy to remember names to map the destination specific tokens to `projectName` variable used as part of client initialization. analytics-backend uses the value of `projectName` to reason where to pipe the event flow.
+The masking helps with preventing abuse and the mapping makes the configuration more flexible to support multiple final destinations.
+Reach out to the data team when in doubt which project to use or if you need to define a new project.
+
+In the final destinations, we separate staging and production environments. We use the same `projectName` value but different tokens to differentiate the production and staging environments. So you only need to update `endpoint` to pick the right destination.
+
+Use the staging environment for testing purposes by setting `endpoint` in the `createClient` initialization to `data.balena-staging.com`. Once you are happy with your tests and ready to go live, update the `endpoint` to `data.balena-cloud.com` for production environment.
+
+Additionally, identify your component using the parameter `componentName`. It's best to keep the value short yet descriptive.
+
+Please ask Operation folks for `#access` to the relevant project in your final destination so you can validate that the tracked events show as expected.
+
+Finally, there is the optional configuration parameter `componentVersion` to track the version of a component sending each event. This is useful to track changes across new component versions so we strongly recommend tracking it.
+
+## Web tracker
+
+[`WebTracker`](src/web.ts) exposes an events tracking interface suitable for web apps. It provides a simple `Page View` event tracking with page loading metrics via `trackPageView`.
+
+When initializing `createWebTracker`, set `prefix` as a short string that describes your project. All events tracked by `WebTracker` will have this `prefix` in their name.
```

Comment:
Right now I guess we use prefix on all web trackers but it's good to keep the prefix optional. On a non web tracker, we already have a case with no prefix: Etcher events. Since Etcher is the only component that sends events to the Etcher project in Amplitude, there is no reason to have a prefix.